types/parse/index.d.ts

```diff
@@ -975,16 +971,23 @@ namespace Parse {
         /**
          * Create a new Schema on Parse
          */
-        save(options?: ScopeOptions): Promise<Schema>;
+        save(): Promise<Schema>;
 
         /**
          * Update a Schema on Parse
          */
-        update(options?: ScopeOptions): Promise<Schema>;
+        update(): Promise<Schema>;
     }
 
     namespace Schema {
-        type TYPE = string | number | boolean | Date | File | GeoPoint | any[] | object | Pointer | Relation;
+        type TYPE = 'String' | 'Number' | 'Boolean' | 'Date' | 'File' | 'GeoPoint' | 'Polygon' | 'Array' | 'Object' | 'Pointer' | 'Relation';
```

Comment:
The official Parse JS doc is wrong. Do not follow it. 🤕 
It may be better to restore. and I create another PR for wrong documentation of Parse JS.
https://github.com/parse-community/Parse-SDK-JS/pull/1071

Reply:
I'm not taking it from the documentation, but from the actual implementation in the SDK: https://github.com/parse-community/Parse-SDK-JS/blob/487bbfb158977a9311a947938b54680538259f1d/src/ParseSchema.js#L15

Reply:
You link is right thing. 
And `index.d.ts` has all types already as GeoPoint, File and so on. So you can use directly.

Reply:
I'm not sure what you mean... the second parameter for `addField()` is not a type but a string!
So one must type, for example:

```ts
schema.addField('defaultFieldString', 'Number');
```
and not:
```ts
schema.addField('defaultFieldString', number);
```

Reply:
Oh. You are right. It will be better to make another one for FieldType only.

How about this? 

```ts
type Type = string | number | boolean | Date | File | GeoPoint | any[] | object | Pointer | Relation;
type FieldType = 'String' | 'Number' | 'Boolean' | 'Date' | 'File' | 'GeoPoint' | 'Polygon' | 'Array' | 'Object' | 'Pointer' | 'Relation'
```

And it will fix another error as below comment.

Reply:
That's already done in line 986. Is that what you mean?

Reply:
> That's already done in line 986. Is that what you mean?
You are already done. I missed it. :)